dpdata/plugins/ase.py

```diff
@@ -112,7 +114,7 @@ def from_multi_systems(
         step: Optional[int] = None,
         ase_fmt: Optional[str] = None,
         **kwargs,
-    ) -> "ase.Atoms":
+    ) -> object:  # generator of "ase.Atoms"
```

Comment:
should use `typing.Generator["ase.Atoms"]`

Reply:
done